sqlglot/parser.py

```diff
@@ -77,6 +77,7 @@ def _parse_decimal(args):
         TokenType.DESC,
         TokenType.ENGINE,
         TokenType.EXPLAIN,
+        TokenType.FALSE,
```

Comment:
is it legal to name a column false or true?

Reply:
yes, it is legal in Spark to name a column `true` `TRUE`